hw03-generics/src/main/java/homework/CustomerService.java

```diff
@@ -0,0 +1,43 @@
+package homework;
+
+import java.util.Map;
+import java.util.SortedMap;
+import java.util.TreeMap;
+
+public class CustomerService {
+
+    public final SortedMap<Customer, String> customers = new TreeMap<>();
```

Comment:
1) private
```suggestion
    private final SortedMap<Customer, String> customers = new TreeMap<>();
```

2) Здесь будет достаточно интерфейса `NavigableMap`